ui/raidboss/data/00-misc/test.ts

```diff
@@ -325,10 +327,36 @@ const triggerSet: TriggerSet<Data> = {
         },
       },
     },
+    {
+      id: 'Test Combatant Cast Enable',
+      type: 'GameLog',
+      netRegex: NetRegexes.echo({ line: 'cactbot test combatant cast.*?', capture: false }),
+      run: (data) => {
+        data.watchingForCast = true;
+      },
+    },
+    {
+      id: 'Test Combatant Cast',
+      type: 'CombatantMemory',
+      netRegex: NetRegexes.combatantMemory({
+        pair: [{ key: 'IsCasting1', value: '1' }, { key: 'CastBuffID', value: '.*?' }],
```

Comment:
I believe they should always show up on a line together, because of the nature of the underlying logic handling in the game client itself, but technically if something changed down the line in the game client and `CastBuffID` no longer always got set back to all `0`s in memory, this would no longer be the case.